Readme.md

```diff
@@ -484,7 +439,41 @@ class OrganizationFinder extends AbstractDoctrineFinder
 }
 ```
 
-... todo: more explanations
+### How to implement you own event listener ?
+
+To implement other custom event listeners you only need to implement the `EventListenerInterface`:
+
+```php
+class ConsoleOutputListener implements EventListenerInterface
+{
+    /**
+     * @var SystemLoggerInterface
+     * @Flow\Inject
+     */
+    protected $systemLogger;
+
+    /**
+     * @param ButtonTagged $event
+     * @param EventMetadata $metadata
+     */
+    public function whenButtonTagged(ButtonTagged $event, RawEvent $rawEventData)
```

Comment:
Yes, the `RawEvent` is there to just show it is an optional second argument. Would probably be good to also show a use-case where it is used.